client/src/Main/ProblemCategoryPage/index.js

```diff
@@ -14,7 +14,14 @@ import ProblemCategoryPage from './view';
 class ProblemCategoryPageContainer extends Component {
   render() {
     return (
-      <Query query={GET_PROBLEM_CATEGORIES_PARETO} fetchPolicy="network-only">
+      <Query
+        query={GET_PROBLEM_CATEGORIES_PARETO}
+        variables={{
+          startDate: null,
+          endDate: null,
+        }}
+        fetchPolicy="network-only"
```

Comment:
ce serait mieux de mettre un "cache-and-network" ici, cf https://www.apollographql.com/docs/react/api/react-apollo#graphql-config-options-fetchPolicy

Reply:
done